docs/CLI.md

```diff
@@ -112,6 +112,58 @@ OPTIONS:
    --comment-syntax value  Comment syntax to use for snapshot files (default: "//")
 ```
 
+## `scip test`
+
+```
+NAME:
+   scip test - Validate subsets of snapshot files
+
+USAGE:
+   scip test [command options] [arguments...]
+
+DESCRIPTION:
+   The test subcommand validates whether a provided scip index
+   file matches manually specified symbol fields within an index. This can be used in
+   conjunction with snapshot to provide a cohesive testing framework for scip indexers.
+
+OPTIONS:
+   --from value            Path to SCIP index file (default: "index.scip")
+   --comment-syntax value  Comment syntax to use for snapshot files (default: "//")
+   --help, -h              show help
+```
+
+### File Specification
```

Comment:
Looks like this file `docs/CLI.md` is used in a unit test to validate the output of `scip --help`

I'm open to suggestions on a better location for this documentation

Reply:
I realize I recommended linking to the `main` branch earlier, but it's probably better to link to the URL for the released version so that there is less risk of confusion in case an old version of the subcommand doesn't support some syntax supported by a new version/on main. I've pushed a change for that.